codex/codex.nim

```diff
@@ -129,7 +129,7 @@ proc bootstrapInteractions(
   return (client, host, validator)
 
 proc start*(s: CodexServer) {.async.} =
-  notice "Starting codex node"
+  notice "Starting codex node", config = $s.config
```

Comment:
is this printing the entire config object?

Reply:
Yes, perhaps we can add a separate `trace/debug` that prints it. It's very useful for debugging.

Reply:
On second thought, this could be useful info for the user. We can always downgrade the log level later if we dislike it.

Reply:
I think it can get somewhat large tho, maybe `trace` is more appropriate?

Reply:
It's not too bad, and shows default values that were not specified by the user. But i can change it to trace.